documentation/book/assembly-cluster-operator.adoc

```diff
@@ -8,16 +8,17 @@
 [id='cluster-operator-{context}']
 = Cluster Operator
 
-{ProductName} uses the Cluster Operator to deploy and manage Kafka (including Zookeeper) and Kafka Connect clusters.
-The Cluster Operator is deployed inside of the Kubernetes cluster.
-To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created within the Kubernetes cluster.
-Based on what is declared inside of the `Kafka` resource, the Cluster Operator deploys a corresponding Kafka cluster.
-For more information about the different configuration options supported by the `Kafka` resource, see xref:assembly-deployment-configuration-kafka-str[]
+{ProductName} uses the Cluster Operator to deploy to a Kubernetes cluster and manage the following:
```

Comment:
This sounds a bit strange to me. I had to read it twice to understand what it is trying to say. I wonder if we can somehow rephrase it to make it more understandable.

Reply:
I've rewritten. I've also moved, as we were saying pretty much the same in the next section. Each operator assembly now has a single line to say what it does. The following concepts give more detail.